h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -438,6 +440,16 @@ public void init(boolean expensive) {
         error("_solver", "Solvers GRADIENT_DESCENT_LH and GRADIENT_DESCENT_SQERR are only " +
                 "supported for ordinal regression.  Do not choose them unless you specify your family to be ordinal");
       switch (_parms._family) {
```

Comment:
When a user specify Family.AUTO, what it mean is we will try to assign a family to the dataset, right?  So, instead of carrying the Family.AUTO, are we going to assign it a family during init()?  For example, if nclass() ==1, we assign Family.Gaussian, nclass() == 2, we assign Family.Binomial, nclass() > 2, we assign multinomial?